api/camunda/camunda-openapi.yaml

```diff
@@ -41,8 +41,8 @@ paths:
     get:
       tags:
         - License
-      summary: Get status of Camunda license
-      description: Obtains the status of the current Camunda license
+      summary: Get status of Camunda License
```

Comment:
This, and many changes like it, are a result of us not making changes upstream yet 😅 

@christinaausley and I opened https://github.com/camunda/camunda/pull/21596 today to fix that. 

@christinaausley, I don't know how you feel, but I would be fine with these regressions making it into this PR knowing that they will be fixed by the next one. Otherwise, we'll want to manually fix these like in this PR like we did in https://github.com/camunda/camunda-docs/pull/4166.